_includes/sections/working-here/your-pay-pension-and-other-benefits.md

```diff
@@ -108,38 +108,100 @@ your holiday allowance shown on your BreatheHR dashboard.
+
+If you are unwell and know that you are going to be off for a period of time, or
+have been signed off by your GP until a certain date, you don’t need to update
+your line manager daily. Let them know when you will be checking in again so
+they know when to expect to hear from you.
+
+When you return to work, you must check your sickness has been correctly
+recorded in BreatheHR, update it if necessary and mark yourself as returned to
+work. Your line manager will then review and close the sick leave. If you're
+sick for more than 7 calendar days in a row, you'll also need to provide a fit
+note.
+
+##### Absence meetings
+
+Your line manager should check-in with you on a regular basis about your
+wellbeing, and catch up with you after any sickness absence to see how you are.
+If you're sick for more than 7 working days over a rolling 12 month period, your
+line manager will invite you to an absence meeting to discuss your health. Where
+appropriate, we’ll talk about how we can support your return to work and any
+temporary or permanent adjustments which might help improve your wellbeing and
+reduce your levels of absence. We might also agree an action plan and / or a
+review period. Where we think it would be helpful, we’ll seek medical advice via
+an Occupational Health advisor.
+
+If you have a long term health condition and would like to discuss reasonable
+adjustments and ways of working that will help, we welcome a conversation at any
+point. The sooner we know, the sooner we can support you.
+
+##### Pay
+
+Within the 12 month period, a person will receive:
+
+- full pay for the first 12 days of sickness absence
+- half pay (but no less than living wage) for the next 8 days of sickness
+  absence
+- living wage for the next 10 days of sickness absence
+- statutory sick pay pay after 30 days of sickness absence
+
+There'll be circumstances where we'll make exceptions, because it's not possible
+to write a policy that is fair in every situation. This structure will form our
+starting point for those decisions. All decisions on exceptions will be made by
+the Directors’ group. Our aim is to ensure that we’re treating everyone fairly
+and consistently.
+
+##### Long term medical treatment
+
+Regular and open communication with your line manager is important to ensuring
+we can support you in the best possible way through a period of long term
+medical treatment (a period of more than 2 weeks).
+
+If you require long term medical treatment that would affect your ability to
+work as normal, but you are still able to work, we will support this wherever we
+can. We may consider a change of project or role where that’s an option.
+
+If you finish your treatment, you’ll have a meeting to aid your return to work,
+or to normal working hours, and ensure your ongoing wellbeing is supported. We
+may look at a phased return for example.
+
+##### Escalation
+
+If your sickness absence continues for 3 months after an absence meeting, your
```

Comment:
What does "continues for 3 months" mean? A solid 3 months off? Some number of days in the next 3 months? A single day in the next 3 months?

Reply:
> What does "continues for 3 months" mean? A solid 3 months off? Some number of days in the next 3 months? A single day in the next 3 months?

I second the question. Flexibility is good, and I am happy to hear that flexibility is the intent, but vagueness creates anxiety.

Would a potential applicant reading the Playbook be reassured that they won't be expected to not miss a single day once they've been "flagged" via an absence meeting?

Reply:
A sustained amount of absence. It would be following any changes that may have been made in the initial absence meeting ie 80% FTE, compressed hours and even following those changes things haven't improved.

Reply:
Can we state that in the policy?

Reply:
This is still really unclear. Without having a discussion with someone, I'd have no idea what "continues for 3 months" means. If it's something agreed with your line manager during your absence review meeting, then let's state that instead.